examples/using-web-socket/main_test.go

```diff
@@ -10,7 +12,12 @@ import (
 )
 
 func Test_WebSocket_Success(t *testing.T) {
-	wsURL := fmt.Sprintf("ws://%s/ws", "localhost:8001")
+	port := testutil.GetFreePort(t)
+	t.Setenv("HTTP_PORT", strconv.Itoa(port))
+	wsURL := fmt.Sprintf("ws://%s/ws", fmt.Sprint("localhost:", port))
```

Comment:
fmt using fmt as argument



```suggestion
	wsURL := fmt.Sprintf("ws://localhost:%d/ws", port)
```